### PR TITLE
Improve internal icon generation

### PR DESCRIPTION
**Pull request by diegoreymendez**
This PR modifies the script that adds the version number to our icons when building for internal releases, so that the icons are not modified in the working directory.

This PR:

- Renames the relevant build phase to "App Icons: Add Version For Internal Releases".
- Moves the phase one step down (needed so that the icons are copied to the intermediate directories before it's run).
- Moves the script to an external file to make it easier to edit and maintain.
- Improves the script so that it can deal with paths with spaces (was breaking).
- This PR also uses the long version number instead of the short one, so we can have a short version number in internal builds too.

**How to test:**

1. Archive an internal release.
2. Make sure the icons in the final product are right

**How to check the icons in the final archive:**

 - Open the organizer
 - Right click on .xcarchive and select "Show in Finder"
 - Go to Products > Applications
 - Right click on "WordPress.App" and select "Show Package Contents".
 - Verify the icons are OK.

### CODE REVIEW

**Comment by elibud** [comment]
@diegoreymendez I won't be able to properly review this for a few days, but the changes look solid. There's one thing we need to test and that is uploading a build generated with this script to HockeyApp and distributing it that way to make sure the icon is displayed correctly. As long as we test that I'm happy for this to go in with @frosty's approval.

**Comment by elibud** [comment]
BTW I ❤️ that you are moving this script out, the diffs with the previous configuration were useless but they will become meaningful now that's an external script.

File: Scripts/BuildPhases/AddVersionToIcons.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    rm /tmp/temp.png
+    rm /tmp/labels-base.png
+    rm /tmp/labels.png
+    rm "${base_tmp_normalizedFilePath}"
+    
+    echo "Overlayed ${final_file_path}"
+}
+
+# Process all app icons and create the corresponding internal icons
+# icons_dir="${SRCROOT}/Images.xcassets/AppIcon.appiconset"
```
Comment (diegoreymendez) [inline Scripts/BuildPhases/AddVersionToIcons.sh]: Self reminder: I should remove this line.

**Comment by diegoreymendez** [comment]
I just added this to the description, which I had forgotten:

```
- This PR also uses the long version number instead of the short one, so we can have a short version number in internal builds too.
```

**Comment by diegoreymendez** [comment]
Thanks for the review @frosty!

I'm merging since @elibud is AFK for now, but do feel free to comment on any of this when you're back.

**Pull request merged**